regolith/helpers/u_milestonehelper.py

```diff
@@ -108,16 +108,15 @@ def db_updater(self):
                 for i in numbered_milestones:
                     current_mil = numbered_milestones[i]
                     del current_mil['identifier']
-                print('Please inform name, objective, and due date to add a new milestone')
-                return
+                raise RuntimeError('Please inform name, objective, and due date to add a new milestone')
```

Comment:
"name, objective, and due date are required for a new milestone"

Reply:
done!